<ui version="4.0" >
 <class>ValuesWidget</class>
 <widget class="QWidget" name="ValuesWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="gbGeneral" >
     <property name="title" >
      <string>General</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QCheckBox" name="chbValuesEnabled" >
        <property name="text" >
         <string>annotate values</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lType" >
        <property name="text" >
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QComboBox" name="cbType" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text" >
          <string>x</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>y</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>z</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="lPlacement" >
        <property name="text" >
         <string>Placement</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QComboBox" name="cbPlacement" >
        <item>
         <property name="text" >
          <string>above</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>under</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>right</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>left</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="lRotation" >
        <property name="text" >
         <string>Rotation</string>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="3" column="1" >
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>123</width>
          <height>24</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2" >
       <widget class="QSpinBox" name="sbRotation" />
      </item>
      <item row="3" column="3" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Â°</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="lDistance" >
        <property name="text" >
         <string>Distance to the line</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" >
       <widget class="QSpinBox" name="sbDistance" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbFormat" >
     <property name="title" >
      <string>Format</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5" >
      <item row="0" column="0" >
       <widget class="QLabel" name="lLabelsPrecision" >
        <property name="text" >
         <string>Precision</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>121</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" >
       <widget class="QSpinBox" name="sbLabelsPrecision" >
        <property name="minimum" >
         <number>0</number>
        </property>
        <property name="maximum" >
         <number>10</number>
        </property>
        <property name="value" >
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lLabelsFormat" >
        <property name="text" >
         <string>Format</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QComboBox" name="cbLabelsFormat" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="lLabelsDateFormat" >
        <property name="text" >
         <string>Time/Date/Datetime format</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLineEdit" name="leLabelsDateFormat" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbText" >
     <property name="title" >
      <string>Text</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="lFont" >
        <property name="text" >
         <string>Font</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="KFontRequester" name="kfrTextFont" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lFontColor" >
        <property name="text" >
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="KColorButton" name="kcbTextColor" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>418</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KFontRequester</class>
   <extends>QWidget</extends>
   <header>kfontrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
